pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -76,3 +76,23 @@ svg {
       }
   }
 }
+
+// PF assumes check sizes are Chrome-based; size needs to be set for cross-browser compatibility
```

Comment:
As always the issue needs to be reported upstream. Of course not the fix, but at least issue with screenshots from the docs which prove it's not our issue.

Reply:
This is reported upstream, with the fix.

I forgot to add the comment in this PR; it's in the issue. I'll push with changes in a minute.